### DISCUSSION TITLE
Disable background anime girls in main menu.

### DISCUSSION THREAD

**Discussion by elenakrittik** [ppy/osu#24892] [Q&A] [answered]
Last time i played lazer these were just random anime pictures. Now i see nothing but, uh, how to say that politely.. "girls with very little clothes on them". No idea where osu takes these pictures from, but i don't want to see this type of background in a rhythm game.

(Screenshots were recorded consecutively, by just opening and closing game. This happens EVERY SINGLE TIME.)

![osu! Screenshot 2023 09 23 - 16 47 58 08](https://github.com/ppy/osu/assets/77104725/04ad161b-e2b3-4d9c-ad2e-41d4fdda2024)
![osu! Screenshot 2023 09 23 - 16 50 28 10](https://github.com/ppy/osu/assets/77104725/358f3fcf-e3be-4c0b-81c2-4bce6aac2e94)
![osu! Screenshot 2023 09 23 - 16 51 08 29](https://github.com/ppy/osu/assets/77104725/39b778ba-e261-49d1-84c6-b7285d99ced9)
![osu! Screenshot 2023 09 23 - 16 51 57 31](https://github.com/ppy/osu/assets/77104725/7afae621-a21b-4812-8889-130cdc8bbc99)

**Comment by Ladwick** [comment, marked as answer]
Go to options and look for an option called "Seasonal Background", then set it to never

**Reply by elenakrittik** [reply]
Thanks!
